<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageWidget</class>
 <widget class="QWidget" name="LanguageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>130</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>523</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>523</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(190, 190, 190);
	border-bottom: 1px solid  rgb(100, 100, 100);
}
</string>
  </property>
  <widget class="QPushButton" name="textFileNameButton">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(150, 150, 150);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: rgb(160, 160, 160);
}

QPushButton:disabled {
	background-color: rgb(190, 190, 190);
}
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locFileNameEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>253</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 10pt &quot;Liberation Mono&quot;;
	color: rgb(20, 20, 20);

	background-color: rgb(255, 255, 255);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="locFileNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 11pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}
</string>
   </property>
   <property name="text">
    <string>Default .loc file name:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="textFileNameEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>253</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Mono</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 10pt &quot;Liberation Mono&quot;;
	color: rgb(20, 20, 20);

	background-color: rgb(255, 255, 255);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}
</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="deletableCheckBox">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>15</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
	background-color: rgb(190, 190, 190);
	border: none;
}

QCheckBox::indicator {
	width: 20px;
	height: 20px;
}

QCheckBox:indicator:checked {
    image: url(:/images/images/checked-checkbox.png);
}

QCheckBox:indicator:unchecked {
    image: url(:/images/images/unchecked-checkbox.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="locFileNameButton">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(150, 150, 150);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: rgb(160, 160, 160);
}

QPushButton:disabled {
	background-color: rgb(190, 190, 190);
}
</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="textFileNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 11pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}
</string>
   </property>
   <property name="text">
    <string>Default .txt file name:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="languageLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 75 11pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}
</string>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="CustomComboBox" name="languageComboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: rgb(150, 150, 150);

	border: 1px solid  rgb(100, 100, 100);
	border-radius: 2px;
}

QComboBox:hover {
	background-color: rgb(160, 160, 160);
}

QComboBox:disabled {
	background-color: rgb(190, 190, 190);
}
</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
   <property name="maxCount">
    <number>100</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
